src/TorchSharp/TorchVision/Ops.cs

```diff
@@ -30,5 +33,48 @@ public static Tensor sigmoid_focal_loss(Tensor inputs, Tensor targets, float alp
                 return loss.MoveToOuterDisposeScope();
             }
         }
+
+        public static Tensor nme(Tensor boxes, Tensor scores, double iou_threshold = 0.5)
+        {
+            using (var nmsScope = torch.NewDisposeScope()) {
+                var x1 = boxes[.., 0];
```

Comment:
System.Range is only available in later version of .NET, and we need to be compatible with .NET FX 4.7.2. Therefore, you need to use the uglier syntax that System.Range turns into.

Reply:
changed it to tensor.select(..)

Reply:
You could also use: 

```C#
var idx = TensorIndex.Slice(); // Default values is the same as '..'
var x1 = boxes[idx, 0];
```

I don't know if that or 'select()' is faster.